collect_app/src/main/java/org/odk/collect/android/widgets/UrlWidget.java

```diff
@@ -105,13 +115,14 @@ protected void onDetachedFromWindow() {
         }
```

Comment:
Please add @Override I know it's not required but if we do that for all the other methods let's keep consistency.